catalog/core/catalog-core-api/src/main/java/ddf/catalog/source/InternalIngestException.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p/>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p/>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+package ddf.catalog.source;
+
+public class InternalIngestException extends IngestException {
```

Comment:
Can you add JavaDoc explaining that this is for internal server errors that should be represented as 5xx instead of 4xx status code.